tests/Functional/Doctrine/Mock/StatementMock.php

```
@@ -21,12 +21,14 @@
 
 class StatementMock implements IteratorAggregate, Statement
 {
-    public function bindValue($param, $value, $type = null)
+    public function bindValue($param, $value, $type = null): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$param'. (undefined)

(UnusedFormalParameter)